### PR TITLE
Use en-US as fallback when using other default language

### LINKED ISSUE #21199: Gitea fails to start because of non-translated string
### Description

On gitea service startup (using systemd):
`Unable to register cron task update_migration_poster_id Error: translation is missing for task "update_migration_poster_id", please add translation for "admin.dashboard.update_migration_poster_id"`
This fails the service start.

I’ve setup gitea with:
```
[i18n]
LANGS = fr-FR
NAMES = Français
```
Adding english makes it work.

I don’t think it should hard fail for a missing translated string…

### Gitea Version

1.17.2

### Operating System

Arch Linux

### How are you running Gitea?

Systemd package using provided systemd service.

### Database

PostgreSQL

### PR DESCRIPTION
**Pull request by wxiaoguang**
Only en-US has complete translations. When use other language as default, the en-US should still be used as fallback.

Close #21199

### Screenshot

image

### CODE REVIEW

File: modules/translation/translation.go [hunk 1]
```diff
@@ -75,9 +75,17 @@ func InitLocales(ctx context.Context) {
 
 		matcher = language.NewMatcher(supportedTags)
 		for i := range setting.Names {
-			key := "locale_" + setting.Langs[i] + ".ini"
+			var localeDataBase []byte
+			if i == 0 && setting.Langs[0] != "en-US" {
```
Comment (lunny) [inline modules/translation/translation.go:79]: en-US could be the second language?

Comment (wxiaoguang) [reply modules/translation/translation.go:79]: Yes, it could. But what do you mean?

Comment (wxiaoguang) [reply modules/translation/translation.go:79]: @lunny

**Review by lunny — APPROVED** [review]

**Comment by wxiaoguang** [comment]
@go-gitea/maintainers need one more review

**Review by lafriks — APPROVED** [review]

**Comment by lafriks** [comment]
please send backport

**Pull request merged**